ApplicationInsights/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -63,7 +61,8 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
     /**
      * Logs that a page or other item was viewed. 
      * @param IPageViewTelemetry The string you used as the name in startTrackPage. Defaults to the document title.
-     * @param customProperties Additional data used to filter events and metrics. Defaults to empty.
+     * @param customProperties Additional data used to filter events and metrics. Defaults to empty. If a user wants
+     *                         to provide a custom duration, it'll have to be in customProperties
```

Comment:
add field name for customers here

Reply:
It's hard to see what is highlighted in GitHub pull requests :(. But what is the field name?